shared/js/background/atb.es6.js

```diff
@@ -30,7 +30,7 @@ var ATB = (() => {
 
                 xhr.onreadystatechange = function () {
                     if (xhr.readyState === XMLHttpRequest.DONE) {
-                        if (xhr.status == 200) {
+                        if (xhr.status === 200) {
```

Comment:
Strict equality here is safe, `xhr.status` is `unsigned short` [according to MDN](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/status).